console/views/store.py

```diff
@@ -100,7 +100,12 @@ def store(page):
     datetime_earliest = request.args.get('datetime_earliest', type=str, default='')
     datetime_latest = request.args.get('datetime_latest', type=str, default='')
 
-    form = StoreForm(tx_id=tx_id)
+    form = StoreForm(
+        tx_id=tx_id,
+        ru_ref=ru_ref,
+        survey_id=survey_id
+    )
+
     if form.validate():
         store_data = get_filtered_responses(
             audited_logger, valid, tx_id, ru_ref, survey_id, datetime_earliest, datetime_latest)
```

Comment:
In the else just under this can we put this instead..
`store_data = []`

Reply:
Sounds good, I'll get that done

Reply:
Fixed